examples/turn-server/simple/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.1"
+
+services:
+  pion-turn:
+    container_name: "pion-turn"
+    image: pion-turn:${VERSION:-latest}
+    build:
+      context: ./
+    stdin_open: true
+    environment:
+      - VERSION=${PION_TURN_VERSION:-master}
+      - REALM=${PION_TURN_REALM:-localhost}
+      - USERS=${PION_TURN_USERS:-username=password}
+      - PUBLIC_IP=${PION_TURN_PUBLIC_IP:-127.0.0.1}
+      - UDP_PORT=${PION_TURN_UDP_PORT:-3478}
+    network_mode: host
+    ports:
+      # STUN
+      - "${PION_TURN_UDP_PORT:-3478}:${PION_TURN_UDP_PORT:-3478}"
```

Comment:
TCP?

Reply:
Yes, could be also added here.

Reply:
was merged without TCP and custom TURN port.
I did not have bandwith to work on it.
I may to another PR later for thoses.

Reply:
No problem. We are not in a rush. I just wanted to have this merged so we can improve it later if somebody finds time to improve this.